app/components/ProjectList.tsx

```diff
@@ -10,9 +13,42 @@ interface ProjectListProps {
 }
 
 export default function ProjectList(props: ProjectListProps) {
+
+  const [currentProjectIndex, setCurrentProjectIndex] = useState(0);
+
+  function wrapProjectIndex(index: number): number {
+
+    if (index <= -1) {
+      return (props.projectList.projects.length - 1);
+    }
+    else if (index >= props.projectList.projects.length) {
+      return 0;
+    }
+    else {
+      return index;
+    }
+
+  }
+
+  function incrementProjectIndex() {
```

Comment:
you can also write functions like this: `const incrementProjectIndex = () => {}`

Reply:
Is there a general guideline on when to use this kind of function declaration (besides when you need the closure)? As a habit I tend to only use arrow functions for anonymous functions, but I'm not sure what the common practice is in JavaScript.